src/utils/table.ts

```diff
@@ -12,36 +24,36 @@ import { Table } from "types/Table";
  * @param titlePrefix what should the tables be called (will have the table number appended to it)
  * @param appendTableNumber whether to append the table number to the title or not
  * @param startFrom what number should we start from when generating table numbers in the title
+ * @param subtitle what should the tables subtitle be
  */
-export const generateTables: (props: {
-  num: number;
-  capacity: number;
-  rows?: number;
-  columns?: number;
-  titlePrefix?: string;
-  appendTableNumber?: boolean;
-  startFrom?: number;
-}) => Table[] = ({
-  num,
-  capacity,
-  rows = DEFAULT_TABLE_ROWS,
-  columns = DEFAULT_TABLE_COLUMNS,
-  titlePrefix = "Table",
-  appendTableNumber = true,
-  startFrom = 1,
-}) =>
-  Array.from(Array(num)).map((_, idx) => {
+export const generateTables = (props: GenerateTablesProps): Table[] => {
+  const {
+    num,
+    capacity,
+    rows = DEFAULT_TABLE_ROWS,
+    columns = DEFAULT_TABLE_COLUMNS,
+    titlePrefix = "Table",
+    appendTableNumber = true,
+    startFrom = 1,
+    subtitle,
+  } = props;
+
+  return Array.from(Array(num)).map((_, idx) => {
     const tableNumber = startFrom + idx;
 
     const title = appendTableNumber
       ? `${titlePrefix} ${tableNumber}`
       : titlePrefix;
 
+    const reference = props.makeReference?.(props) ?? title;
```

Comment:
out of curiosity, why haven't you added `makeReference` to the destructuring assignment list of the `props`?

Reply:
Not for any good reason that I can remember off the top of my head. May have just written it that way originally and not cleaned it up or similar.